beaconChain/constants/constants.ts

```diff
@@ -1,58 +1,52 @@
-// https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#constants
-
 // Misc
```

Comment:
I don't know if this PR is appropriate for this, but it would be nice to import/export constants and enums from constants/index.ts like we're now doing with types/index.ts.

Reply:
Hmm.. I like it
Let's do next PR and make an issue.